source/deployment/elasticsearch.rst

```diff
@@ -0,0 +1,73 @@
+Elasticsearch Beta (E20)
+========================
+
+*Available in Enterprise Edition E20.*
+
+`Elasticsearch <https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html>`_ is a distributed, RESTful search engine capable of supporting search in large scale enterprise deployments with millions of messages. 
```

Comment:
Hm, does this imply MM doesn't support search in deployments with millions of messages? `capable of supporting search in large scale enterprise deployments with millions of messages.`

Maybe we can say instead something like below, so it's more clear why E20 customers would benefit from it.

```
`Elasticsearch <https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html>`_ is a distributed, RESTful search engine supporting highly efficient database searches in a `cluster environment <https://docs.mattermost.com/deployment/cluster.html>`_.
```